main/ui/src/main/java/org/cryptomator/ui/util/ListenerRegistry.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2014 Sebastian Stenzel
```

Comment:
Put your own name in both places. If I modify this class in a major way, I'll add myself to the contributors list.